modules/extensions/src/views/lite/components/debugger/index.tsx

```diff
@@ -50,6 +50,12 @@ export class Debugger extends React.Component<Props, State> {
       onClick: this.toggleDebugger
     })
 
+    this.props.store.view.addCustomAction({
+      id: 'actionDebug',
+      text: 'debug message',
+      select: this.handleSelect
+    })
```

Comment:
To keep consistency, can you use 'label' for the text, and onClick instead of select ?